lib/ecs_deploy/instance_fluctuation_manager.rb

```diff
@@ -132,16 +134,36 @@ def extract_target_container_instances(decrease_count, az_to_container_instances
       target_container_instances
     end
 
-    def stop_tasks(arn)
-      running_task_arns = ecs_client.list_tasks(cluster: @cluster, container_instance: arn).task_arns
+    # list tasks whose desired_status is "RUNNING" or
+    # whoose desired_status is "STOPPED" but last_status is "RUNNING" on the ECS container
+    def list_running_task_arns(container_instance_arn)
+      running_tasks_arn = ecs_client.list_tasks(cluster: @cluster, container_instance: container_instance_arn).flat_map(&:task_arns)
+      stopped_tasks_arn = ecs_client.list_tasks(cluster: @cluster, container_instance: container_instance_arn, desired_status: "STOPPED").flat_map(&:task_arns)
+      stopped_running_task_arns = stopped_tasks_arn.each_slice(MAX_DESCRIBABLE_ECS_TASK_COUNT).flat_map do |arns|
+        ecs_client.describe_tasks(cluster: @cluster, tasks: arns).tasks.select do |task|
+          task.desired_status == "STOPPED" && task.last_status == "RUNNING"
+        end
+      end.map(&:task_arn)
+      running_tasks_arn + stopped_running_task_arns
+    end
+
+    def wait_until_tasks_stopped(task_arns)
+      @logger.info("All old tasks: #{task_arns.size}")
```

Comment:
You removed the word "old" from the method name in https://github.com/reproio/ecs_deploy/pull/65/commits/0f540bb80bd356e686fd874e90d33b5e0060bca7, but does the word still remain here and in the last log on purpose?